src/mmw/js/src/draw/utils.js

```diff
@@ -72,6 +72,13 @@ function placeMarker(map, drawOpts) {
     return defer.promise();
 }
 
+function createRwdMarkerIcon(iconName) {
```

Comment:
Consider refactoring this further to remove the duplicate `L.marker` and `bindPopup` code
```
function createRwdMarker(latlng, iconName, popupMessage) {
   return L.marker(latlng, {
        icon : L.divIcon({
            className: 'marker-rwd marker-rwd-' + iconName,
            iconSize: [16,16]
        })
    }).bindPopup(popupMessage);
}
```